### PR TITLE
add linked data to live blogs

### PR DESCRIPTION
**Pull request by johnduffell**
Live blogs could do with being marked up with the liveblog schema for better SEOness.  This adds most of the fields we already have info for.

Unfortunately this doesn't validate yet due to the schema.org for live blogs not being up yet so the sainty tests will be failing =(  I will try to hack them to get around this problem.

### CODE REVIEW

File: common/app/model/content.scala [hunk 1]
```diff
@@ -429,6 +429,8 @@ private object ArticleSchemas {
     // http://schema.org/Review
     if (article.isReview)
       "http://schema.org/Review"
+    else if (article.isLive)
+      "http://schema.org/LiveBlogPosting"
```
Comment (robertberry) [inline common/app/model/content.scala]: This 404s

**Comment by robertberry** [comment]
:+1: otherwise

File: common/app/model/content.scala (hunk 1 above)
Comment (robertberry) [inline common/app/model/content.scala]: Assume you wanted `BlogPosting`

Comment (johnduffell) [inline common/app/model/content.scala]: I did mention it doesn't validate due to the schema.org not being up yet :laughing:

Comment (robertberry) [inline common/app/model/content.scala]: Oh yeah. OOPS

**Comment by johnduffell** [comment]
thanks @robertberry !

**Pull request merged**